mobile.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="/mobile.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="main-nav">
+            <div class="status-bar">
+                <div class="time-style">
+                    <p class="time">9:14</p>
+                </div>
+                <div class="group-4">
+                    <div class="group">
+                        <div class="cellular">
+                            <img src="icons/CellularConnection.png" alt="">
+                        </div>
+                        <div class="wifi">
+                            <img src="icons/Wifi.png" alt="">
+                        </div>
+                        <div class="battery">
+                            <img src="icons/Battery.png" alt="">
+                        </div>
+                    </div>
+                   
+                </div>
+            </div>
+            <div class="tool-bar">
+                <div class="logo">
+                    <p>akinlade temitope</p>
+                </div>
+                <div class="normal-button">
+                    <div class="icon">
+                        <img src="icons/Union.png" alt="">
+                    </div>
+                </div>
+            </div>
+
+            
+        </nav>
+    </header>
```

Comment:
- [ ] This section is expected to have a background image just as the one provided in the design. Please ensure you refer to the Figma design. It is highly important.
- [ ] Please ensure to add the margins provided in the design. 
- [ ] Please ensure that you use the font and colors provided in the Figma design. One of the most important abilities of a developer is to ensure that they follow the design provided.